recipes-debian/whois/whois_debian.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "This client is intelligent and can automatically select the appropriate whois server for most queries."
```

Comment:
whois package and inetutils package cannot be installed in same time. Please fix it in case someone need both package. It's okay to give higher priority to whois package than inetutils one.

```
WARNING: core-image-minimal-1.0-r0 do_rootfs: Postinstall for package inetutils failed with 1:
update-alternatives: Error: not linking /home/masami/emlinux/build/tmp-glibc/work/qemuarm64-emlinux-linux/core-image-minimal/1.0-r0/rootfs/usr/bin/whois to /usr/bin/whois.inetutils since /home/masami/emlinux/build/tmp-glibc/work/qemuarm64-emlinux-linux/core-image-minimal/1.0-r0/rootfs/usr/bin/whois exists and is not a link   
```

Reply:
I fixed it.

If only the whois package is installed, it will be used.
```
root@qemuarm64:~# ls -la /usr/bin/whois*
lrwxrwxrwx 1 root root    20 Oct 26 02:59 /usr/bin/whois -> /usr/bin/whois.whois
-rwxr-xr-x 1 root root 85672 Oct 26 02:51 /usr/bin/whois.whois
```
```
root@qemuarm64:~# whois --version
Version 5.4.3.

Report bugs to <md+whois@linux.it>
```

If installed with inetutils package, the commands of the whois package will take precedence.
```
root@qemuarm64:~# ls -la /usr/bin/whois*
lrwxrwxrwx 1 root root    20 Oct 26 02:53 /usr/bin/whois -> /usr/bin/whois.whois
-rwxr-xr-x 1 root root 71864 Oct 26 02:20 /usr/bin/whois.inetutils
-rwxr-xr-x 1 root root 85672 Oct 26 02:51 /usr/bin/whois.whois
```
```
root@qemuarm64:~# whois --version
Version 5.4.3.

Report bugs to <md+whois@linux.it>.
```
```
root@qemuarm64:~# whois.whois --version
Version 5.4.3.

Report bugs to <md+whois@linux.it>.
```
```
root@qemuarm64:~# whois.inetutils --version
whois (GNU inetutils) 1.9.4
Copyright (C) 2015 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Marco d'Itri.
```

Reply:
Thanks.